VA-Explorer/va_explorer
# Add loading indicator for initial dashboard load

This MR adds an additional loading indicator to cover the whole row of charts in addition to the individual charts to prevent user confusion when presented with loaded but empty charts

To verify:
1. Pull, runserver, login
2. Navigate to dashboard (assumes mock data already loaded)
3. Observe lack of empty charts while data finishes loading, continue to observe when tweaking various filters on dashboard

Closes #104
